### PR TITLE
feat: rich text widget upload image

### PR DESCRIPTION
**Pull request by MeisamMahdian**
Its part of the rich text widget release 2.2.0

### CODE REVIEW

**Comment by ConnorLand** [comment]
Hi @MeisamMahdian . What's the release date for 2.2.0?

**Comment by MeisamMahdian** [comment]
> Hi @MeisamMahdian . What's the release date for 2.2.0?

Hi @ConnorLand it's not decided yet. I will let you know when it's known.

**Comment by ConnorLand** [comment]
> > Hi @MeisamMahdian . What's the release date for 2.2.0?
> 
> Hi @ConnorLand it's not decided yet. I will let you know when it's known.

Sounds good. I'll get this PR text ready now. If you just let me know 1-2 days before the release, I'll be sure to merge the PR on release day. Thanks!

File: content/en/docs/appstore/widgets/rich-text.md [hunk 1]
```diff
@@ -159,7 +159,17 @@ To configure this widget, follow these steps:
...
+    * **No** (default) – If selected, the **upload** tab is made unavailable in the image dialog box.
+    * **Yes** – If selected, the **upload** tab below becomes available in the image dialog box. This enables the end-user to upload an image to the Mendix server.
 
+        {{< figure src="/attachments/appstore/widgets/rich-text/upload-image.png" alt="upload-image" >}}
+
+        * **Upload Image Endpoint** – This sets the endpoint that receives the image entity as the body parameter and passes the image GUID in return.
+        * **Upload Image Max Size** – This controls the maximum allowed image size (in KB).
+        * **Uploaded Image List** – This is a list of images that has been used as content in a comma-separated format.
+        
+        <!-- Note: See this module for easy setup of the image upload functionalities such as the endpoint [Rich Text Editor Extension Module]( // TODO: module link ) -->
```
Comment (ConnorLand) [inline content/en/docs/appstore/widgets/rich-text.md:172]: We'll be sure not to merge until this module link is in place.

**Comment by MariaShaposhnikova** [comment]
@MeisamMahdian , @ConnorLand , this PR has been open for over a year, is it still relevant?

**Comment by MeisamMahdian** [comment]
Closed. It's out of scope now.

**Pull request closed without merging**